test/helpers/pagelets_helper_test.rb

```diff
@@ -1,17 +1,17 @@
 require 'test_helper'
+require 'statics_isolator_helper'
 
 class PageletsHelperTest < ActionView::TestCase
+  include Isolator
   include PageletsHelper
 
+  isolate_config Pagelets::Manager
```

Comment:
The problem is that instead of removing just the pagelets that were added this test clears all of them. The `add_pagelet` part should just be in a setup block, and if needed separated in several contexts.